src/components/views/voip/PipView.tsx

```diff
@@ -54,6 +54,16 @@ const SHOW_CALL_IN_STATES = [
     CallState.WaitLocalMedia,
 ];
 
+/**
+ * Callback to create the pip content view.
+ * Passes down the resize and move callbacks to the child component.
```

Comment:
> Passes down the resize and move callbacks to the child component.

I don't really understand what this means.

Is it saying "things that implement this type should pass down the callbacks to the child component"? The implementations you've written doesn't seem to pass the callbacks anywhere, so are they incorrect implementations?

Reply:
I moved the comments to the referenced interface: https://github.com/matrix-org/matrix-react-sdk/pull/9604/files#diff-0015e3d8911aaf58cc5ac9696c660082f35c5027983a1ed1bd84960dad2887e6R36-R39

Hope they are not better :neutral_face: